doc/TSCScope.md

```diff
@@ -0,0 +1,31 @@
+Subject to the contraints defined in the foundation
+charter, these are the areas that the TSC has autonomy
+in making decisions (some of which can be delegated to
+other groups, for example the CTC).  Outside of these
+areas the TSC will collaborate with the board
+when making decisions:
+
+* Code/Doc/ changes to any of the repositories under github/nodejs
+* Standards and processes covering contributions to repositories under
```

Comment:
Is the intention that working groups would be covered by this clause? Some of them certainly make commits on the repo (docs, diagonistics, etc) some are more process oriented (inclusivity WG), but does this cover everything? I'm thinking of cases like the evangelism WG.